src/globus_cli/commands/timer/create.py

```diff
@@ -0,0 +1,288 @@
+import datetime
+from typing import List, Optional, Tuple, cast
+
+import click
+from globus_sdk import TimerJob, TransferData
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import command, group, mutex_option_group
+from globus_cli.termio import FORMAT_TEXT_RECORD, formatted_print
+
+from ._common import (
+    DATETIME_FORMATS,
+    JOB_FORMAT_FIELDS,
+    START_HELP,
+    parse_timedelta,
+    read_csv,
+    require_activated_endpoints,
+)
+
+INTERVAL_HELP = """
+    Interval at which the job should run. Use 'w', 'd', 'h', 'm', and 's' as suffixes
+    to specify weeks, days, hours, minutes, and seconds. Examples: '1h 30m', '500s',
+    '24h', '1d 12h', '2w', etc. Must be in order: hours -> minutes -> seconds. You
+    should either use quotes ('1d 2h') or write without spaces (1d2h).
+"""
+
+
+@group("create", short_help="Submit a Timer job", hidden=True)
+def create_command():
+    pass
```

Comment:
Not strictly necessary at the moment, but at some point I'd like to move this to make `create/` a subpackage and `create/transfer.py` the implementation.

Very very early in CLI development we tried some other structures for things, but this is the one which we found was intelligible to people because the layout (on disk) mirrors the usages.